### DISCUSSION TITLE
adding theme key throws error unless dev server is restarted

### DISCUSSION THREAD

**Discussion by dkebler** [squidfunk/mkdocs-material#5895] [Q&A]
Just trying to follow the docs.  Can't even get off off square one.

when I add theme key to yaml an error is throw on render.

```
site_name: Docs
theme:
  name: material
```

but then if I restart the dev server the error is not thrown, nor if I then continue to add keys under `theme`.   Maybe docs should indicate adding 'theme:` might require dev server restart.

**Comment by squidfunk** [comment]
We're happy to look into it. Could you create a [bug report](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/) for this, following our process?

Discussions are for questions, not for bug reports 😉
